lib/comms/http_driver.ex

```diff
@@ -46,8 +46,17 @@ defmodule Onfido.Comms.HttpDriver do
     ]
   end
 
-  defp options do
-    [ssl: [{:versions, [:"tlsv1.2"]}]]
+  defp default_ssl_options() do
+    [
```

Comment:
These don't need to be in the `ssl` key any more?

Reply:
I don't think so - I've bumped poison up from 1.x to 2.x and followed [this](test/kraken/studios/schedules/schedule_updater_test.exs) migration guide